apps/chat_server/lib/chat_server/schemas/message.ex

```diff
@@ -10,6 +10,8 @@ defmodule ChatServer.Schema.Message do
     belongs_to :room, Schema.Room
     belongs_to :user, Schema.User
 
+    many_to_many :users_following, Schema.User, join_through: "starred_messages, on_delete: :delete_all, on_replace: :delete"
```

Comment:
The long string value passed to `join_through` feels wrong here. Tried to find an example in the definitive documentation but haven't found one yet.

I would expect the syntax to be:

```elixir
many_to_many :users_following, Schema.User, join_through: "starred_messages", on_delete: :delete_all, on_replace: :delete
```

Since `on_delete` and `on_replace` should be keyword list options, not string values that need to be parsed an interpolated.

Reply:
Also, it feels like there's a better name for this relationship. The term "Following" could mean a number of things. For instance, what if we introduce threaded replies to messages. A user who has replied once might be "following" that message.

What about a self-documenting name like `:starred_message_users`. It's a little verbose, but it would prevent any hesitation about the value.